setup.cfg

```
@@ -46,6 +47,10 @@ install_requires =
     click
     anymarkup
```

Can you please add `ogr` and `jsonschema`?

Thanks!
```suggestion
     anymarkup
     jsonschema
     ogr
```